src/OrchardCore.Commerce.Web/appsettings.Development.json

```diff
@@ -5,5 +5,9 @@
       "Microsoft": "Warning",
       "Microsoft.Hosting.Lifetime": "Information"
     }
+  },
+  "Stripe": {
+    "ApiKey": "sk_test_51H59owJmQoVhz82aOUNOuCVbK0u1zjyRFKkFp9EfrqzWaUWqQni3oSxljsdTIu2YZ9XvlbeGjZRU7B7ye2EjJQE000Dm2DtMWD",
+    "PublishableKey": "pk_test_51H59owJmQoVhz82aWAoi9M5s8PC6sSAqFI7KfAD2NRKun5riDIOM0dvu2caM25a5f5JbYLMc5Umxw8Dl7dBIDNwM00yVbSX8uS"
```

Comment:
Don't share these keys.

Reply:
These two are [publicly available keys](https://stripe.com/docs/keys#:~:text=TYPE-,VALUE,-WHEN%20TO%20USE), I just used them for testing. Later I will replace them. What would you recommend for hiding these in the future?

Reply:
Environment variables or user secrets. See [_Safe storage of app secrets in development in ASP.NET Core_](https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-6.0).

I think both of those take precedence over the JSON file in the configuration builder, so it shouldn't hurt to keep these in. But you should include a comment explaining that they are the test API keys, including the link you mentioned.